test/multilib/fpus.test

```diff
@@ -0,0 +1,26 @@
+# RUN: %clang -print-multi-flags-experimental --target=arm-none-eabihf -march=armv7m -mfpu=fpv5-d16 | FileCheck --check-prefix=VFPV4-D16 %s
+# RUN: %clang -print-multi-flags-experimental --target=arm-none-eabihf -march=armv7m -mfpu=fpv5-d16 | FileCheck --check-prefix=FPV4-SP-D16 %s
+# RUN: %clang -print-multi-flags-experimental --target=arm-none-eabihf -march=armv7m -mfpu=fpv5-d16 | FileCheck --check-prefix=FPV5-SP-D16 %s
```

Comment:
```suggestion
# RUN: %clang -print-multi-flags-experimental --target=arm-none-eabihf -march=armv7m -mfpu=fpv5-d16 | FileCheck --check-prefixes=VFPV4-D16,FPV4-SP-D16,FPV5-SP-D16 %s
```

Reply:
I tried this and it didn't work very well. The problem is that you have to arrange the checks in the right order: if you expect `-mfpu=fpv4-sp-d16` to appear in the output before `-mfpu=vfpv4-d16` then the two check lines have to be in that order. But the order of output from `-print-multi-flags[-experimental]` is not guaranteed, so that introduces fragility – if the output order changes, the test can break in a way that isn't actually interesting. With the test as I have it now, it's insensitive to the output ordering.

Separately from this, we discussed offline this morning rewriting the generation code in Python rather than cmake for legibility. I'm still planning to do that.

(Indeed, that could be a thing that changes the output order, because who knows what order the Python version will find it most convenient to generate!)

Reply:
I think you should assume that the order of `-print-multi-flags[-experimental]` is guaranteed. Many tests in llvm-project would break if it wasn't. See https://reviews.llvm.org/D142933#change-zuw0O32XbYXO

Reply:
Ah, they're sorted by clang? I must admit I'd assumed that they would be emitted in whatever order `multilib.yaml` generated them, which in turn would depend on the generating script and its inputs. But fine, if clang is guaranteeing to sort them into a canonical order, then that's much easier.